build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -483,7 +483,7 @@ CONFIG_SPI_EXCHANGE=y
 # CONFIG_GPIO is not set
 CONFIG_I2S=y
 # CONFIG_AUDIO_DEVICES is not set
-CONFIG_BCH=y
+# CONFIG_BCH is not set
```

Comment:
Could you tell me how this feature affect the file system?

Reply:
CONFIG_BCH enables ramdisk testcases, but it is not certain now that ramdisk feature can be enabled in NXP boards.